### ISSUE TITLE
unexpected output from visualize utility when circuit contains blackbox

### ISSUE THREAD

**Issue by erblum** [circuitgraph/circuitgraph#32]
Calling the visualize utility on a circuit containing a blackbox produced an unintuitive output; I thought I would mention it here in case it is not working as intended.

Example code:

> import circuitgraph as cg
> my_circuit = cg.Circuit()
> my_blackbox = cg.BlackBox("blackbox",["in_0","in_1"],["out_0","out_1"])
> my_circuit.add_blackbox(my_blackbox,"my_bb")
> for i in [0,1]:
>     my_circuit.add(f"in_{i}", "input")
>     my_circuit.add(f"out_{i}", "output")
>     my_circuit.connect(f"in_{i}",f"my_bb.in_{i}")
>     my_circuit.connect(f"my_bb.out_{i}",f"out_{i}")
> cg.visualize(my_circuit,<file path>)

I expected the image to show the blackbox connected to inputs on the left and outputs on the right. In the output I got (below) the outputs are not connected to anything and there are additional nodes shown.

![testfile](https://user-images.githubusercontent.com/26673728/164793446-cf16469e-9886-446c-835e-329c6c11a19d.jpg)

**Comment by rbnprdy** [comment]
This should be fixed in the new release (0.2.0)

```python
import circuitgraph as cg
my_circuit = cg.Circuit()
my_blackbox = cg.BlackBox("blackbox", ["in_0", "in_1"], ["out_0", "out_1"])
my_circuit.add_blackbox(my_blackbox, "my_bb")
for i in [0, 1]:
    my_circuit.add(f"in_{i}", "input")
    # in 0.2.0, `output` is no longer a separate type. Instead, any node can be marked as an output.
    my_circuit.add(f"out_{i}", "buf", output=True) 
    my_circuit.connect(f"in_{i}", f"my_bb.in_{i}")
    my_circuit.connect(f"my_bb.out_{i}", f"out_{i}")
cg.visualize(my_circuit, "out.png")
```

Gives

![out](https://user-images.githubusercontent.com/20175683/164935564-bd93af90-7999-459d-995d-9ae38806f058.png)

**Comment by erblum** [comment]
Working on my end now, too. Thanks for taking care of that so fast!
